engines/python/setup/djl_python/rolling_batch/vllm_rolling_batch.py

```diff
@@ -149,6 +150,20 @@ def inference(self, input_data: list[str], parameters: list[dict]) -> list:
                     f"Beam search is not supported yet, use first output by default"
                 )
             self.request_cache[req_id]["finished"] = request_output.finished
+            # Record SD metrics
+            completion_output = request_output.outputs[0]
+            if self.vllm_configs.record_acceptance_rate and request_output.finished and completion_output.acceptance_history:
+                record = {}
```

Comment:
This only dumps the metrics to the console right? Looks good to me.

Reply:
yes